internal/account/account.go

```diff
@@ -0,0 +1,10 @@
+package account
+
+import "time"
+
+type Account struct {
+	ID        int       `json:"id"`
```

Comment:
Why use int for ID?

Reply:
sorry I missed this one, willl correct it in the next PR